pkg/redhat/setup.sh

```diff
@@ -37,10 +37,10 @@ curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | sudo tee /etc/yu
 echo "Installing build pre-requisites..."
 yum groupinstall -y "Development Tools"
 
-if [ "${OS_VERSION}" == 7 ]; then
-    yum install -y expect fakeroot httpd-devel postgresql15-devel python3-devel nodejs yarn rpm-build rpm-sign yum-utils krb5-devel
-    pip3 install sphinx
-    pip3 install sphinxcontrib-youtube
+if [ "${OS_VERSION}" == 8 ]; then
```

Comment:
This should also be properly quoted to avoid comparing a string and an int (I realise the old code was also broken).